azure-pipelines.yml

```diff
@@ -223,3 +228,5 @@ stages:
 #       railsEnv: 'production'
```

Comment:
could we remove all this bunch of comments 🤔 ?

Reply:
I think this is the code for deploying our production app which we'll have at some point. Best to ask @tpinney.